node/l1-public-v2.yml

```diff
@@ -48,14 +48,34 @@ paths:
               $ref: '#/components/schemas/Transaction'
       responses:
         200:
-          description: $DAG transaction has been received for processing
+          description: Hash of $DAG transaction that has been received for processing
```

Comment:
It's a description of the status (endpoint), not the response. So 200 Ok means that the DAG transaction has been accepted for processing

Reply:
Shouldn't we also describe the body somewhere? 🤔

Reply:
We return a string, that's a hash but we don't mention it anywhere.

Reply:
Probably we can put a description for the response itself

Reply:
I will check it out.